app/records/views.py

```
@@ -67,11 +77,52 @@ def record_detail_view(request, id):
         record=record,
     )
 
+    determine_delivery_options = True
+
     if record.custom_record_type:
         if record.custom_record_type == "ARCHON":
+            determine_delivery_options = False
             template_name = "records/archon_detail.html"
         if record.custom_record_type == "CREATORS":
             template_name = "records/creator_detail.html"
+            determine_delivery_options = False
+
+    if determine_delivery_options:
+        # Only get the delivery options if we are looking at records
+        # Get the delivery options for the iaid
+        delivery_options_context = {}
+
+        try:
+            delivery_options = delivery_options_request_handler(
+                iaid=record.iaid
+            )
+
+            delivery_options_context = construct_delivery_options(
+                delivery_options, record, request
+            )
+
+        except Exception as e:
+            # Built in order exception option
+            logger.error(
+                f"DORIS Connection error using url '{os.getenv("DELIVERY_OPTIONS_API_URL", "")}' - returning OrderException from Availability Conditions {str(e)}"
+            )
```

Can you add sentry logging for this silent error. This link was shared in our discussions - https://docs.sentry.io/platforms/python/usage/#capturing-messages

---

Added